docs/cli/testkube_pro_docker.md

```diff
@@ -0,0 +1,50 @@
+
+<head>
+  <meta name="og:type" content="reference-doc" />
+</head>
+
+## testkube pro docker
+
+Run Testkube Docker Agent and connect to Testkube Pro environment
+
+```
+testkube pro docker [flags]
+```
+
+### Options
+
+```
+      --agent-prefix string             usually don't need to be changed [required for custom cloud mode] (default "agent")
+      --agent-token string              Testkube Pro agent key [required for centralized mode]
+      --agent-uri string                Testkube Pro agent URI [required for centralized mode]
+      --api-prefix string               usually don't need to be changed [required for custom cloud mode] (default "api")
+      --docker-container string         Docker container name for Testkube Docker Agent (default "testkube-agent")
+      --docker-image string             Docker image for Testkube Docker Agent (default "kubeshop/testkube-agent:2.1.39")
+      --env-id string                   Testkube Pro environment id [required for centralized mode]
```

Comment:
i didn't think env-id and org-id were required?

Reply:
they are not required for docker container, but we use them to login to cloud and store api tokeen in cli cloud context. I used the same apparoach like we have in pro init. but can remove Login block, if it's not necessary

Reply:
can we just mark them as optional?

Reply:
they are optional and not used in case of 
`--no-login testkube set context   Ignore login prompt, set existing token later by testkube set context`
I can add adjust docs

Reply:
thank you!